### PR TITLE
Added page titles

### LINKED ISSUE #42: Set page titles
Titles are currently just Impactasaurus | Impactasaurus. Change to be {page title} | Impactasaurus. We currently have react-helmet configured with a page title template of `%s | Impactasaurus`, but lower level pages aren't setting their sub titles.

### PR DESCRIPTION
**Pull request by nikw92**
Closes #42

### CODE REVIEW

File: src/app/containers/Account/index.tsx [hunk 1]
```diff
@@ -37,15 +38,20 @@ class Account extends React.Component<any, IState> {
...
-        <Grid.Column>
-          <h1>Account</h1>
-          <h3>Change Password</h3>
-          {this.renderChangePassword()}
-          <h3>Delete Account</h3>
-          <p>To delete your account, please send an email to <a href="mailto:support@impactasaurus.org?Subject=DeleteAccount">support@impactasaurus.org</a></p>
-        </Grid.Column>
-      </Grid>
+      <div>
+        <Helmet>
```
Comment (drimpact) [inline src/app/containers/Account/index.tsx]: Any reason these couldn't sit under the `Grid` rather than wrapping again?

**Comment by drimpact** [comment]
Missing titles for `OutcomeSet` and `Review`. Although in the normal flow of the application you would always go via `OutcomeSets` and `ReviewSelector`, if a link is shared, users may get taken straight to `OutcomeSet` or `Review`.

File: src/app/containers/Account/index.tsx (hunk 1 above)
Comment (nikw92) [reply src/app/containers/Account/index.tsx]: I was getting the 'JSX elements must have a single parent error while editing the pages that did not have `Grid` so I wrapped their contents in a `div` and used the same approach for the rest of the pages. But if you'd like the `Helmet` tags under `Grid` then I'll make the necessary changes.

**Comment by drimpact** [comment]
That would be great, just gets rid of the unnecessary wrappingOn 5 Sep 2017 5:06 p.m., nikw92 <notifications@github.com> wrote:@nikw92 commented on this pull request.



In src/app/containers/Account/index.tsx:
> @@ -37,15 +38,20 @@ class Account extends React.Component<any, IState> {
 
   public render() {
     return (
-      <Grid container columns={1} id="Account">
-        <Grid.Column>
-          <h1>Account</h1>
-          <h3>Change Password</h3>
-          {this.renderChangePassword()}
-          <h3>Delete Account</h3>
-          <p>To delete your account, please send an email to <a href="mailto:support@impactasaurus.org?Subject=DeleteAccount">support@impactasaurus.org</a></p>
-        </Grid.Column>
-      </Grid>
+      <div>
+        <Helmet>

I was getting the 'JSX elements must have a single parent error while editing the pages that did not have Grid so I wrapped their contents in a div and used the same approach for the rest of the pages. But if you'd like the Helmet tags under Grid then I'll make the necessary changes.

**Comment by nikw92** [comment]
Made the requested changes

**Pull request merged**